App/Sagas/NodeOnline.ts

```diff
@@ -27,5 +28,8 @@ export function * onNodeOnline () {
 
     // Check for new photos on every online event
     yield put(StorageActions.refreshLocalImagesRequest())
+
+    // update
+    yield call(getSDKVersion)
```

Comment:
This was better in `NodeStarted` like you had it before

Reply:
Better because if there is some problem and the node never comes online, we won't be able to see the node version while looking into the problem